APCAppCore/APCAppCore/DataSubstrate/Model/APCUser+Bridge.m

```diff
@@ -81,6 +92,44 @@ - (void)signUpWithDataGroups:(NSArray<NSString *> *)dataGroups onCompletion:(voi
     }
 }
 
+- (void) signUpWithDataGroups:(NSArray<NSString *> *)dataGroups
```

Comment:
Can we have the old method signature still exist, just have it call this one with vc = nil? Then we don't break existing apps before we have a chance to update them.